src/vue/fields/DateField.vue

```diff
@@ -13,19 +13,16 @@
     </label>
 
     <div class="date-field-flatpickr__field">
-      <flat-pickr
-        v-bind="$attrs"
-        v-on="$listeners"
+      <input
+        type="text"
+        ref="dateField"
         class="date-field-flatpickr__input"
-        :config="config"
+        @input="dateFieldUpdated"
         v-model="flatpickrDate"
+        :disabled="disabled"
         :placeholder="placeholder"
         :key="_uid"
```

Comment:
Not, because when flatpickr build DOM-model of the Date-Field, it looks for this input attr and set appropriate state.

But to change it dynamically, we need to watch for `disabled` prop changes and change state directly. Provided this functional in the new commits iteration

Reply:
`:key` check